app/src/main/java/org/mozilla/fenix/settings/RadioButtonPreference.kt

```diff
@@ -75,44 +94,49 @@ class RadioButtonPreference @JvmOverloads constructor(
 
     private fun updateRadioValue(isChecked: Boolean) {
         persistBoolean(isChecked)
-        radioButton.isChecked = isChecked
+        radioButton?.isChecked = isChecked
         Settings.getInstance(context).preferences.edit().putBoolean(key, isChecked)
             .apply()
     }
 
     private fun bindRadioButton(holder: PreferenceViewHolder) {
         radioButton = holder.findViewById(R.id.radio_button) as RadioButton
-        radioButton.isChecked = Settings.getInstance(context).preferences.getBoolean(key, false)
-        radioButton.setStartCheckedIndicator()
+        radioButton?.isChecked = Settings.getInstance(context).preferences.getBoolean(key, false)
+        radioButton?.setStartCheckedIndicator()
     }
 
     private fun toggleRadioGroups() {
-        if (radioButton.isChecked) {
+        if (radioButton?.isChecked == true) {
             radioGroups.forEach { it.updateRadioValue(false) }
         }
     }
 
     private fun bindTitle(holder: PreferenceViewHolder) {
-        val titleView = holder.findViewById(R.id.title) as TextView
+        titleView = holder.findViewById(R.id.title) as TextView
 
         if (!title.isNullOrEmpty()) {
-            titleView.text = title
+            titleView?.text = title
         }
     }
 
     private fun bindSummaryView(holder: PreferenceViewHolder) {
         summaryView = holder.findViewById(R.id.widget_summary) as TextView
 
         if (!summary.isNullOrEmpty()) {
-            summaryView.text = if (shouldSummaryBeParsedAsHtmlContent) {
+            summaryView?.text = if (shouldSummaryBeParsedAsHtmlContent) {
```

Comment:
nit: put all of this inside a `?let` or `?apply` call since this function is a no-op when `summaryView` is null